apps/metadata-editor-e2e/src/e2e/edit.cy.ts

```diff
@@ -585,7 +585,9 @@ describe('editor form', () => {
         })
         it('should add a keyword', () => {
```

Comment:
Could it make sense to add a test, which checks that the input is cleared on selection?